Story-Chief/frontend-test-pieter-jan-scheir
# Altered hook

## Description
Changed the event type filter (hook) function so that if multiple filters are applied. The events that contain one or more types are visible in the calendar.


## Closes issue(s)
N/A


## Testing Instructions
Select multiple event types in the dropdown and verify if the correct items are visible in the calendar.





## Screenshots
N/A
